hmy/api_backend.go

```diff
@@ -220,6 +220,16 @@ func (b *APIBackend) GetPoolTransactions() (types.PoolTransactions, error) {
 	return txs, nil
 }
 
+// GetAccountNonce returns the nonce value of the given address for the given block number
+func (b *APIBackend) GetAccountNonce(
+	ctx context.Context, address common.Address, blockNr rpc.BlockNumber) (uint64, error) {
```

Comment:
No need to worry about the pending transactions anymore?

Reply:
There is getPoolNonce that does the same job

Reply:
I mean to say if there is a pending transaction shouldn't you be returning that number instead of just the committed transactions?

Reply:
ah i see. now understood. yeah, I guess we do need to consider pending txns here.

Reply:
@gupadhyaya actually, isn't pool nonce >= account nonce always? so getPoolNonce() is used for getting number of transactions including pending, and getAccountNonce() used for getting number of confirmed transactions?

Reply:
what will happen if from the cli I made a transaction which is waiting in the pool and then I made the next transaction? Both of them will have the same nonce and only one will work?